### ISSUE TITLE
sequence  coordinate files

### ISSUE THREAD

**Issue by ValWood** [pombase/website#565]
- [x]  CDS_coordinates, by chromosome - ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/CDS_Coordinates/
- [x]  exon coordinates - ftp://ftp.pombase.org/pombe/Exon_Coordinates/

**Comment by ValWood** [comment]
these appear to be generated already....

**Comment by kimrutherford** [comment]
> these appear to be generated already....

The data files in the pombe directory on the FTP site are copied from the old FTP site.  They aren't generated.

**Comment by kimrutherford** [comment]
The nightly load now generates these coordinate files.  From tomorrow they'll appear here: ftp://ftp.pombase.org/nightly_update/misc/

**Comment by kimrutherford** [comment]
> ftp://ftp.pombase.org/nightly_update/misc/

I've noticed a difference between the PomBase v1 files and the new ones.  In V1 the exon files include the UTR exons but the new files don't.  Which behaviour is best?

**Comment by ValWood** [comment]
I guess for exon coordinates all exons coding and non coding should be included if they were before....
Does everyone agree?

**Comment by mah11** [comment]
no problem with it, especially if that's how we'd been doing it (and nobody complained)

**Comment by kimrutherford** [comment]
I've changed the exporter to include the UTR exons.  The file now mostly looks like the file that Ensembl produced.  The differences mostly look like small coordinate tweaks.

I noticed one difference that's a bit tricky, SPAC1071.01c was:

    SPAC1071.01c    3855385 3857994 -1

and is now:

    SPAC1071.01c   3855385 3855790 -1
    SPAC1071.01c   3855792 3857994 -1

it looks an artificial frame shift has been introduced in the contig file.  Because it's now stored as two exons in the contig file, it has two exons in Chado.  So there are two exons in the exon coord file.  That will be the case for any other genes with a curated frame shirt.

Is that OK?  We could come up with a rule for merging the exons when writing the coord file but I'm a bit uneasy about that.
